common/changes/@microsoft/rush/sennyeya-allow-cobuild-orchestration_2024-08-08-23-11.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Adds a new experiment for cobuilds to allow uncacheable operations to benefit from cobuild orchestration without using the build cache.",
```

Comment:
Include the name of the experiment.

Reply:
How is this intended to be used? Do both the env var and the option in `rush-project.json` need to be set, or does the env var implicitly set the `rush-project.json` option to `true` for all projects?

If it isn't a conventional experiment (in `experiments.json`) the changlog message should probably be revised.

Reply:
added to environments.json in [3b5d30b](https://github.com/microsoft/rushstack/pull/4871/commits/3b5d30be8c4fd801d6a7c4dabd6c553a6532533c)